lib/elastic/enterprise-search/client/permissions.rb

```diff
@@ -0,0 +1,30 @@
+# Document Permissions API - /documentation/enterprise-search/api/document-permissions
+module Elastic
+  module EnterpriseSearch
+    class Client
+      module Permissions
+
+        def list_all_permissions(content_source_key, current: 1, size: 20)
```

Comment:
```suggestion
        def list_all_permissions(content_source_key, current: 1, size: 25)
```

Reply:
25 is the default pageSize
![image](https://user-images.githubusercontent.com/11838280/66422654-71a38a00-e9e0-11e9-8184-79f368fd80ef.png)